hack/deploy-logging-marketplace.sh

```diff
@@ -205,6 +205,19 @@ if [ "$USE_CUSTOM_IMAGES" = true ] ; then
     oc set env deploy/cluster-logging-operator --list | grep _IMAGE=
 fi
 
+### HACK !!!
+# Once this pr/304 is applied, we should remove this HACK.
+# https://github.com/operator-framework/community-operators/pull/304
+clrole=$( oc get roles --selector olm.owner.kind=ClusterServiceVersion -o name )
+clroleyaml=$( mktemp )
+oc get $clrole -o yaml > $clroleyaml
+if [ grep -q -e "- prometheusrules" $clroleyaml ]; then
```

Comment:
omit the `[` here.  so just
```bash
if grep -q -e "- prometheusrules" $clroleyaml ; then
```
note that in bash, the `[` is really an alias for the `test` command - it does not function as a grouping operator as in a real programming language - and the closing `]` is just syntactic sugar to confuse you into thinking otherwise.

Reply:
Sorry, my mistake...  Fixing it.